docs/docsite/rst/community/create_pr_quick_start.rst

```diff
@@ -0,0 +1,269 @@
+  Some collections do not have integration tests. In this case, unit tests are required.
+
+All integration tests are stored in ``tests/integration/targets`` subdirectories.
+Go to the subdirectory containing the name of the module you are going to change.
+For example, if you are fixing the ``mysql_user`` module in the ``community.mysql`` collection,
+its tests are in ``tests/integration/targets/test_mysql_user/tasks``.
+
+The ``main.yml`` file holds test tasks and includes other test files.
+Look for a suitable test file to integrate your tests or create and include a dedicated test file.
+You can use one of the existing test files as a draft.
+
+When fixing a bug, write a task which reproduces the bug from the issue.
+
+Put the reported case in the tests, then run integration tests with the following command:
+
+.. code-block:: bash
+
+  $ ansible-test integration name_of_test_subdirectory --docker -v
+
+For example, if the tests files you changed are stored in ``tests/integration/targets/test_mysql_user/``, the command is as follows:
+
+.. code-block:: bash
+
+  $ ansible-test integration test_mysql_user --docker -v
+
+You can use the ``-vv`` or ``-vvv`` argument, if you need more detailed output.
+
+In the examples above, the default test image is automatically downloaded and used to create and run a test container.
+Use the default test image for platform independent integration tests such as those for cloud modules.
+
+If you need to run the tests against a specific distribution, see the :ref:`list of supported container images <test_container_images>`. For example:
+
+.. code-block:: bash
+
+  $ ansible-test integration name_of_test_subdirectory --docker fedora35 -v
+
+.. note::
+
+  If you are not sure whether you should use the default image for testing or a specific one, skip the entire step - the community can help you later. You can also try to use the collection repository's CI to figure out which containers are used.
+
+If the tests ran successfully, there are usually two possible outcomes:
+
+- If the bug has not appeared and the tests have passed successfully, ask the reporter to provide more details. It may not be a bug or can relate to a particular software version used or specifics of the reporter's local environment configuration.
+- The bug has appeared and the tests has failed as expected showing the reported symptoms.
+
+Fix the bug
+=============
+
+See :ref:`module_contribution` for some general guidelines about Ansible module development that may help you craft a good code fix for the bug.
+
+Test your changes
+=================
+
+1. Run ``flake8`` against a changed file:
+
+  .. code-block:: bash
+
+    $ flake8 path/to/changed_file.py
+
+  It is worth installing (``pip install flake8``, or install the corresponding package on your operating system) and running ``flake8`` against the changed file(s) first. It shows unused imports, which is not shown by sanity tests (see the next step), as well as other common issues.
```

Comment:
Would it make sense to put this at the beginning of step 1, ie mention how to install `flake8` before giving the command?